build/podman/src/test/java/io/quarkus/ts/docker/PodmanBuildIT.java

```diff
@@ -15,6 +16,7 @@
 
 @QuarkusScenario
 @EnabledIfEnvironmentVariable(named = "DOCKER_HOST", matches = ".*podman.*")
+@DisabledIfEnvironmentVariable(named = "PODMAN_AVAILABLE", matches = "false")
```

Comment:
This test is already disabled on Docker (see the line above). We need to exclude the whole module from the build

Reply:
@fedinskiy  so do you mean it's not needed any change in our test suite, just it will work with the Jenkins job change?

Reply:
Ok, I think with my last Jenkins job modification and the exclusion in the general POM of the whole module from the build I did in my last change with profile `exclude.podman` could be worked.